src/main/g8/build.sbt

```
@@ -52,7 +52,7 @@ lazy val `$name;format="norm"$-stream-impl` = (project in file("$name;format="no
   )
   .dependsOn(`$name;format="norm"$-stream-api`, `$name;format="norm"$-api`)
 
-val lombok = "org.projectlombok" % "lombok" % "1.16.18"
+val lombok = "org.projectlombok" % "lombok" % "1.16.20"
```

```suggestion
val lombok = "org.projectlombok" % "lombok" % "1.18.8"
```

Prefer `1.18.8` (which is latest _and_ jdk11 compatible)

---

Oh, this is part of the `1.4.15` release. I don't think lombok `1.16.x` or `1.18.x` will make any difference.

---

Should we update this for the Lagom 1.4.x seed?